functions.php

```diff
@@ -554,3 +554,21 @@ function split_text( $text, $length = 100, $parts = 2 ) {
 
 	return $result;
 }
+
+add_filter( 'gform_ip_address', 'cloudflare_gform_ip_address' );
+/**
+ * Get the visitor IP from CloudFlare
+ *
+ * @param mixed $ip IP address.
+ */
+function cloudflare_gform_ip_address( $ip ) {
+	// Return the IP address provided by CloudFlare.
+	if ( isset( $_SERVER['CF-Connecting-IP'] ) && ! empty( $_SERVER['CF-Connecting-IP'] ) ) {
+		$ip = isset( $_SERVER['CF-Connecting-IP'] );
```

Comment:
That code returns true false rather than IP address ... really confused why do we need it